Yoast/wordpress-seo
# Stories/implement webpack

## Summary

**Warning:** This is meant as a additional build method and should not be used in production before it is tested rigorously.

This PR can be summarized in the following changelog entry:

- Added additional tooling for javascript bundling, resulting in approximately 20% smaller js bundles.

## Relevant technical choices:

* Added webpack and webpack-grunt packages
* Added production configuration for webpack

## Test instructions

This PR can be tested by following these steps:

* Run `grunt build-webpack:js` (grunt alias) or `grunt webpack:build`(webpack build only).
**Note:** `grunt build-webpack-js` will also copy required js dependencies into the `js/dist` folder.
* **Optional:** You can also run the build using webpack only using `yarn run build`.
* Creates javascript bundles in `js/dist`, named the same as the output of `grunt build:js`.
* Make sure wordpress-seo is running in a wordpress instance and see if everything works as expected
